### PR TITLE
DRILL-5766: Fix XSS vulnerabilities in Drill

### PR DESCRIPTION
**Pull request by arina-ielchiieva**
1. Bumped up freemarker version to 2.3.26-incubating.
2. Indicated default output format in Freemarker configuration (HTML).
3. Fixed Web UI bugs listed in DRILL-5346, DRILL-5341, DRILL-5339, DRILL-5338.

Details in [DRILL-5766](https://issues.apache.org/jira/browse/DRILL-5766).

### CODE REVIEW

**Comment by parthchandra** [comment]
+1. Looks good.

File: exec/java-exec/src/main/resources/rest/profile/profile.ftl [hunk 1]
```diff
@@ -162,8 +226,7 @@
       </div>
       <div id="fragment-overview" class="panel-collapse collapse">
         <div class="panel-body">
-          <svg id="fragment-overview-canvas" class="center-block"></svg>
```
Comment (kkhatua) [inline exec/java-exec/src/main/resources/rest/profile/profile.ftl]: @arina-ielchiieva  is there a reason we don't display this canvas any more? Doesn't seem related to the list of bugs fixed.

Comment (arina-ielchiieva) [reply exec/java-exec/src/main/resources/rest/profile/profile.ftl]: With this canvas, fragment overview was displayed weirdly. I have removed so the fragment overview looks the same as others.

Comment (arina-ielchiieva) [reply exec/java-exec/src/main/resources/rest/profile/profile.ftl]: Talked to Kunal, it turned out that we need this canvas to display graphic for fragments execution time, so I guess we need to return it back.
